tests/post_training/test_quantize_conformance.py

```diff
@@ -234,11 +237,175 @@ def quantize_ov_native(model: ov.Model,
     return quantized_model
 
 
-def run_ptq_timm(data, output, model_name, model_quantization_params, process_connection): # pylint: disable=W0703
+def quantize_torch_ptq(model: torch.nn.Module,
+                       calibration_dataset: nncf.Dataset,
+                       preset: nncf.QuantizationPreset = nncf.QuantizationPreset.PERFORMANCE,
+                       target_device: nncf.TargetDevice = nncf.TargetDevice.ANY,
+                       subset_size: int = 300,
+                       fast_bias_correction: bool = True,
+                       model_type: Optional[nncf.ModelType] = None,
+                       ignored_scope: Optional[nncf.IgnoredScope] = None) -> NNCFNetwork:
```

Comment:
`pt_impl_experimental` returns `torch.nn.Module`, not `NNCFNetwork`.

Reply:
Actually PTQ uses `NNCFNetwork` to retrieve `NNCFGraph` and to add quantization modules. `NNCFNetwork`  is returned from `pt_impl_experimental` as well 
![image](https://user-images.githubusercontent.com/74656388/220589951-ffcf5ded-e20c-456b-90f7-3c4408193021.png)

Reply:
Okay, I see. But then, it means that the output type hint for `quantize_impl_experimental` is not correct? It's `torch.nn.Module`, not `NNCFNetwork`.

Reply:
You are right. BTW it's fair for TORCH backend as well

Reply:
Done